ray-operator/config/samples/ray-cluster.heterogeneous.yaml

```diff
@@ -24,8 +24,8 @@ spec:
     # the following params are used to complete the ray start: ray start --head --block --redis-port=6379 ...
     rayStartParams:
       port: '6379'
-      object-manager-port: '12345'
```

Comment:
Can you explain the reason of removing these two lines?

Reply:
These args are certainly optional, unnecessary, and distracting -- I definitely think it's a good idea to remove them, along with the commented out versions on lines 27 and 28.

Reply:
@kevin85421 @DmitriGekhtman  Thanks for the comment. I have deleted these two lines.